crates/viewer/re_renderer/src/queueable_draw_data.rs

```diff
@@ -19,10 +19,10 @@ pub enum QueueableDrawDataError {
 
 type DrawFn = dyn for<'a, 'b> Fn(
         &Renderers,
-        &'b GpuRenderPipelinePoolAccessor<'b>,
+        &GpuRenderPipelinePoolAccessor<'a>,
```

Comment:
Could these lifetimes be given better names?

Reply:
yes! That would be `'pipelines` and `'pass`